### ISSUE TITLE
Step up limit

### ISSUE THREAD

**Issue by Bender2k** [Rem0o/FanControl.Releases#1783]
Hi,

as known, FanControl has a fan speed "Step up" maximum limit of 50%/s.
But what's the reason for that?

My case fans are triggered by the GPU temp and should spin at the lowest possible speed.

With other tools i found out, that the minimum fan speed is 62% where they always start so spin when the 62% is the inital value.

But with the 50%/s rate, this is not the case. It often works, but sometimes -depending on the position- they move a few millimeters and get stuck and cant overcome the motor resistance. Therefore i have to use a fan speed of 70% which makes a difference in noise.

Would it be possible to allow a step up value of 100%/s, so that i can reliably use a fan speed of 62%?

**Comment by Rem0o** [comment]
I think you misunderstood the parameters.

The step is the rate a which the % change when a fan curve dictates a new value. So instead of going 10 => 100, it will do 10, 60, then 100.  But it will get to 100. It can also go to every % in between. It's the MAXIMUM step speed, not the prescribed speed.

62% seems to be the minimum % at which you get spin. I would do 70% start% to give it a kick to make sure it starts, and 62 - 1% as stop%.

TLDR: 

Step up: anything you want
Step down: anything you want
start%: 70%
stop%: 1 below the minimum speed you want, so 61%

**Comment by Bender2k** [comment]
Thank you, you are right.
Your solution works!

I was always thinking to go straight to 62 without a kick.
